modules/videostab/include/opencv2/videostab/stabilizer.hpp

```diff
@@ -77,6 +77,9 @@ class CV_EXPORTS StabilizerBase
     void setFrameSource(Ptr<IFrameSource> val) { frameSource_ = val; }
     Ptr<IFrameSource> frameSource() const { return frameSource_; }
 
+    void setMaskSource(Ptr<IFrameSource> val) { maskSource_ = val; }
```

Comment:
Please use `const Ptr<IFrameSource>&` here.

Reply:
This also breaks the style in stabilizer.hpp, e.g.
`void setFrameSource(Ptr<IFrameSource> val) { frameSource_ = val; }`
`void setMaskSource(Ptr<IFrameSource>& val) { maskSource_ = val; }`

Reply:
I can change it no problem, just wanted to point that out

Reply:
Right way is using of "const reference" (in all these cases).

> to point that out

:+1: